### PR TITLE
Ran Prussin 2019 through mgs-pipeline.

### PR DESCRIPTION
**Pull request by simonleandergrimm**

### CODE REVIEW

File: bioprojects/PRJNA525405/metadata/create_prussin_metadata.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import re
+import csv
+from collections import defaultdict
+sra_run_path = "sraruninfo.csv"
+disease_tsv_path = "biosample.txt"
```
Comment (jeffkaufman) [inline bioprojects/PRJNA525405/metadata/create_prussin_metadata.py:7]: Document where these files came from and how you got them.

File: bioprojects/PRJNA525405/metadata/create_prussin_metadata.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+with open(disease_tsv_path, mode='r') as infile:
+    lines = infile.readlines()
+    biosample_metadata = defaultdict(list)
+    current_bio_sample = None
+    next_line_is_description = None
+    for line in lines:
+        if 'BioSample:' in line:
+            current_biosample = line.split('BioSample:')[1].split(';')[0].strip()
+        if 'season' in line:
+
```
Comment (jeffkaufman) [inline bioprojects/PRJNA525405/metadata/create_prussin_metadata.py]: nit: blank line

File: dashboard/sample_metadata_classifier.py [hunk 3]
```diff
@@ -301,5 +301,16 @@ def interpret(project, papers, bits):
                 location="Copenhagen",
                 date="2015-04",
                 collection="saliva")
+    elif project in papers["Prussin 2019"]["projects"]:
+        sample, na_type, date, description, season = bits
```
Comment (jeffkaufman) [inline dashboard/sample_metadata_classifier.py]: If you're calling this sampling_range in the final dict, call it sampling_range all the way through

**Review by jeffkaufman — APPROVED** [review]

File: bioprojects/PRJNA525405/metadata/create_prussin_metadata.py (hunk 1 above)
Comment (simonleandergrimm) [reply bioprojects/PRJNA525405/metadata/create_prussin_metadata.py:7]: done

File: bioprojects/PRJNA525405/metadata/create_prussin_metadata.py (hunk 2 above)
Comment (simonleandergrimm) [reply bioprojects/PRJNA525405/metadata/create_prussin_metadata.py]: done

File: dashboard/sample_metadata_classifier.py (hunk 3 above)
Comment (simonleandergrimm) [reply dashboard/sample_metadata_classifier.py]: done

**Pull request merged**